code/modules/reagents/newchem/drug.dm

```diff
@@ -0,0 +1,369 @@
+#define SOLID 1
+#define LIQUID 2
+#define GAS 3
+
+#define REM REAGENTS_EFFECT_MULTIPLIER
+
+datum/reagent/nicotine
+	name = "Nicotine"
+	id = "nicotine"
+	description = "Slightly reduces stun times. If overdosed it will deal toxin and oxygen damage."
+	reagent_state = LIQUID
+	color = "#60A584" // rgb: 96, 165, 132
+	overdose_threshold = 35
+	addiction_threshold = 30
+
+datum/reagent/nicotine/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	var/smoke_message = pick("You can just feel your lungs dying!", "You feel relaxed.", "You feel calmed.", "You feel the lung cancer forming.", "You feel the money you wasted.", "You feel like a space cowboy.", "You feel rugged.")
```

Comment:
decide on whether you want this on the reagent itself or picked when it rolls to show